wao/notifier.py

```diff
@@ -30,8 +30,10 @@ def post_request(text, is_from_429_watcher=False):
     print(str(result))
 
     if BrainConfig.IS_429_FIX_ENABLED:
-        if str(result) != TELEGRAM_RESPONSE_200:
+        if str(result) != TELEGRAM_RESPONSE_200 and is_from_429_watcher:
```

Comment:
1. you need to remove IS_429_FIX_ENABLED variable in both repos and increase the version in config by 0.01 
2. from the logs the error was a 400, currently we are handling a 400 like a 429, which will not work. i.e Queueing a 400 and sending it later will not fix, thus change the condition to : if result==429 in both repos and add back the else print inorder to log other error codes.

Reply:
as I mentioned on the previous pr here 429 does not happen under any circumstance but instead other errors occur when overlapping of message occur hence I used not 200 and also the is_from_429_watcher important here because the deleting part crashes the system. but I have removed the write to file if not is_from_429_watcher and instead logged the result

Reply:
Re. 1 - There was no response or code change regarding this. Comment still stands.
Re. 2 - We don't want to handle 400 as 429. That will cause an infinite loop. Comment still stands. Please schedule a call if this comment is not clear. Thanks.

Reply:
@Bemhreth please do not unresolve a comment before fixing - the action is to schedule a call from the previous comment.

Reply:
both 1 and 2 comments have not been applied.